index.html

```diff
@@ -5,11 +5,53 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet" />
+
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="main-header">
-      <a href=""><logo>Noel Bryant</logo></a>
-    </header>
+    <nav class="navbar">
+      <!-- <div class="toolbar"> -->
+      <div class="logo">
+        <a href="">
+          <img src="assets/images/logo.png" alt="my awesome logo" />
+        </a>
+      </div>
+      <div class="tertiary-enabled">
+        <img
+          src="assets/images/mobile-menu.svg"
+          alt="menu icon"
+          class="mobilenav"
+        />
+        <!-- </div> -->
+      </div>
+    </nav>
```

Comment:
- [ ] I appreciate🎉🎉 your efforts in incorporating the `<nav>` tag into your project. To adhere to HTML5 structural guidelines, it is recommended to place the `<nav>` element inside the `<header>` tag. 
- [ ] This ensures a more semantic and organized structure for your HTML document. By following this practice, you can enhance the overall code structure and maintain compliance with HTML5 standards.🤔🤔🤔
- [ ] For more information please [check it](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure)
- [ ] Please use logos width and height so that it will fix the below picture issue 
- [ ] Your decision 
![Screenshot 2023-06-03 143457](https://github.com/NoelLincoln/mportfolio/assets/118761303/a4f3cf09-97fa-42ff-997b-8988c3ebaa98)